addon/components/besluit-plugin/besluit-plugin-card.js

```diff
@@ -0,0 +1,58 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { getTitleForDecision } from '../../utils/besluit-plugin/get-title-for-decision';
+import {
+  insertTitle,
+  insertArticle,
+} from '@lblod/ember-rdfa-editor-lblod-plugins/commands/besluit-plugin';
+
+export default class BesluitPluginCardComponent extends Component {
+  @tracked hasTitle = true;
+  @tracked disableArticleInsert = true;
+
+  constructor(parent, args) {
+    super(parent, args);
+  }
+
+  get controller() {
+    return this.args.controller;
+  }
+
+  @action
+  insertArticle() {
+    insertArticle(this.controller, '', '');
+  }
+
+  @action
+  insertTitle() {
+    insertTitle(this.controller, '');
+  }
+
+  @action
+  selectionChangedHandler() {
```

Comment:
Same as the comment above, I think this can be replaced by a besluit getter, you can then additionally add besluitUri and hasTitle getters